app/views/pages/api_release_notes.html.haml

```diff
@@ -2,6 +2,47 @@
 
 %section.api-documentation
 
+  %hr
+  %h2 19 April 2018
```

Comment:
**[not a blocker]** Maybe a 💭 for a possible feature. Seems like if we had some structured release notes (in either DB or even YAML notation) we could more easily updated them, where this template would just recursively render all of that information.

💡Something to thing about...